db/database.go

```diff
@@ -1819,15 +1819,17 @@ func (db *DatabaseCollectionWithUser) resyncDocument(ctx context.Context, docid,
 				if updatedExpiry != nil {
 					updatedDoc.UpdateExpiry(*updatedExpiry)
 				}
-
 				doc.SetCrc32cUserXattrHash()
 				raw, rawXattr, err = updatedDoc.MarshalWithXattr()
 				return raw, rawXattr, deleteDoc, updatedExpiry, err
 			} else {
 				return nil, nil, deleteDoc, nil, base.ErrUpdateCancel
 			}
 		}
-		_, err = db.dataStore.WriteUpdateWithXattr(key, base.SyncXattrName, db.userXattrKey(), 0, nil, nil, writeUpdateFunc)
+		opts := &sgbucket.MutateInOptions{
+			MacroExpansion: macroExpandSpec(base.SyncXattrName),
+		}
```

Comment:
Is this `DefaultMutateInOpts()` ?

Reply:
I intended DefaultMutateInOpts() to be test-only, as I thought the non-test usages should be more intentional about what they are setting.  I think I'd prefer to leave as-is for now, and possibly revisit during future HLV work that makes the default more complex.